pkg/api/api.go

```diff
@@ -33,14 +36,23 @@ import (
 //	@license.name				GPL-2.0-only
 //	@license.url				https://github.com/fossology/LicenseDb/blob/main/LICENSE
 //
-//	@host						localhost:8080
 //	@BasePath					/api/v1
 //
 //	@securityDefinitions.apikey	ApiKeyAuth
 //	@in							header
 //	@name						Authorization
 //	@description				Token from /login endpoint
+
+const DEFAULT_PORT = 8080
+
 func Router() *gin.Engine {
+
+	if os.Getenv("PORT") != "" {
+		docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%s", os.Getenv("PORT"))
+	} else {
+		docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%d", DEFAULT_PORT)
+	}
```

Comment:
Seems fine, a better way would be to remove the else completely. And making this logic concise. 

```suggestion
  port := os.Getenv("PORT")
	if len(port) == 0 {
	  port = DEFAULT_PORT
	}
  docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%s", port)
```